docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -331,186 +329,261 @@ using System.Threading.Tasks;
+    ```
+    $ cd path-to-your-test-project
+    $ dotnet test
+    ```
 
-    An example `project.json` testing the library called "Library" might look like this:
+And that's it!  You can now test your library across all platforms across all platforms using command line tools.  To continue testing now that you have everything set up, testing your library is very simple:
 
-    ```javascript
-    {
-    	"commands": {
-    		"test":"xunit.runner.dnx"
-    	},
-    	"frameworks": {
-    		"dnxcore50":{
-    			"dependencies": {
-    				"Library":"",
-    				"System.Runtime":"4.0.0-rc1-*",
-    				"xunit":"2.1.0",
-    				"xunit.runner.dnx": "2.1.0-rc1-*"
-    			}
-    		}
-    	}
-    }
-    ```
+1. Make changes to your library.
+2. Run tests from the command line, in your test directory, with `$ dotnet test`.
 
-4. Grab the dependencies for the `rc1` version of `xunit.runner.dnx` by opening a command prompt, navigating to the root of your library, and typing the following:
+Your code will be automatically re-built when you invoke `$ dotnet test`.
 
-    `$ dnu restore -s "https://myget.org/F/xunit"`
-    
-    Due to the pre-release nature of creating libraries right now, you need to get the test runner for Xunit from the Xunit prerelease fed.  This will not be necessary in the future when `xunit.runner.dnx` has a release version placed in NuGet.
-   
-5. Grab all other dependencies across the project:
+Just remember to run `$ dotnet restore` from the command line any time you add a new dependency and you'll be good to go!
 
-    ```$ dnu restore```
+### Structuring a Solution
 
-6. Navigate to your test project and run tests with `dnx test`:
+Another important aspect of multi-project solutions is establishing a good overall project structure. To structure a multi-project library, you must use top-level `/src` and `/test` folders:
 
-    ```
-    $ cd path-to-test-project
-    $ dnx test
-    ```
+```
+/AwesomeLibrary
+|__global.json
+|__/src
+   |__/AwesomeLibrary.Core
+      |__Source Files
+      |__project.json
+   |__/AwesomeLibrary.CSharp
+      |__Source Files
+      |__project.json
+   |__/AwesomeLibrary.FSharp
+      |__Source Files
+      |__project.json
+/test
+   |__/AwesomeLibrary.Core.Tests
+      |__Test Files
+      |__project.json
+   |__/AwesomeLibrary.CSharp.Tests
+      |__Test Files
+      |__project.json
+   |__/AwesomeLibrary.FSharp.Tests
+      |__Test Files
+      |__project.json
+```
+
+The `global.json` file for this solution would look like this:
+
+```json
+{
+    "projects":[ "src", "test"]
+}
+```
 
-And that's it!  You can now test your library across all platforms across all platforms using command line tools.  To continue testing now that you have everything set up, testing your library is very simple:
+This way follows the pattern current project templates in the `dotnet new` command establish, where all projects are placed under a `/src` directory and all tests are placed under a `/test` directory.
```

Comment:
way -> approach

Suggestion:
This approach follows the same pattern established by project templates created with the `dotnet new` command, where all projects are placed under a `/src` directory and all tests are placed under a `/test` directory.